milestone-3/frontend/src/App.tsx

```diff
@@ -0,0 +1,45 @@
+import Navbar from "./components/navbar";
+import {useState, useEffect} from 'react';
+import Home from "./components/home";
+import About from "./components/about";
+import './App.css';
+//import recipes from "./recipeData.json"
+import RecipePage from "./components/recipePage";
+import {
+    BrowserRouter as BrowserRouter, Routes, Route } from 'react-router-dom';
```

Comment:
If you could put this on one line so it's easier to look at that's usually a good idea. Also you're renaming BrowserRouter to BrowserRouter.. which doesn't do anything..